hummingbot/market/kraken/kraken_market.pyx

```diff
@@ -384,6 +398,7 @@ cdef class KrakenMarket(MarketBase):
         for trading_pair, rule in asset_pairs_dict.items():
             try:
                 base, quote = KrakenMarket.split_to_base_quote(trading_pair)
+                base = KrakenMarket.convert_from_exchange_symbol(base)
```

Comment:
this is going to mean that ETH is represented as "ETH" if it's the base asset and as "XETH" if it's the quote asset. I think we would want to convert both base and quote here?

Reply:
Only the base is needed in this section. The base is required to get the `min_order_size` that is needed by `TradingRule`

Reply:
yes you're right